roles/taca/templates/taca_delivery.yml.j2

```diff
@@ -0,0 +1,85 @@
+log:
+    file: "{{ ngi_pipeline_logs }}/taca.log"
+
+deliver:
+    rootpath: "/proj/{{ ngi_pipeline_sthlm_delivery }}/nobackup/NGI/analysis_ready"
```

Comment:
From this I presume we need two versions of taca_delivery.yml deployed? One for Ua and one for Sthlm.